### DISCUSSION TITLE
Is there a Python demo for this

### DISCUSSION THREAD

**Discussion by nuel000** [ZFC-Digital/puppeteer-real-browser#236] [General]
Just wanted to know if there's a python demo

**Comment by RubberArchind** [comment]
Since this is a Node.js library/package, the answer is simply no.

I just found the rebrowser Python implementation. This could be a good starting point for you:
[rebrowser-playwright-python](https://github.com/rebrowser/rebrowser-playwright-python)

**Reply by nuel000** [reply]
Thanks so much @RubberArchind
